homeassistant/components/velbus/config_flow.py

```diff
@@ -31,14 +31,16 @@ def _create_device(self, name: str, prt: str):
         """Create an entry async."""
         return self.async_create_entry(title=name, data={CONF_PORT: prt})
 
-    def _test_connection(self, prt):
+    async def _test_connection(self, prt):
         """Try to connect to the velbus with the port specified."""
         try:
-            controller = velbus.Controller(prt)
-        except Exception:  # pylint: disable=broad-except
+            controller = velbusaio.controller.Velbus(prt)
+            await controller.connect(True)
+            await controller.stop()
+        except Exception as excep:  # pylint: disable=broad-except
```

Comment:
We should catch a more specific exception first. Broad exceptions should only be used as fallback for unknown errors. The library should raise a library specific exception when the connection fails.